src/monitord/main.c

```diff
@@ -162,6 +162,13 @@ int main(int argc, char **argv)
         merror_exit(CONFIG_ERROR, cfg);
     }
 
+    /* Setting default values and reading global configuration */
+    Config.agents_disconnection_time = 20;
+    Config.alert_agent_disconnection_time = 2;
+    if (ReadConfig(CGLOBAL, cfg, &Config, NULL) < 0) {
```

Comment:
Cant't we use an unique config struct with multiple masks like analysisd does?
c |= CREPORTS
c |= CGLOBAL

Reply:
Now that **mond** has a __Config global_ struct field, a second call to **ReadConfig()** is necessary, just like analysisd does